### PR TITLE
[WIP] Prague 2020 full website

### PR DESCRIPTION
**Pull request by thatdocslady**
Full website PR, still missing the following:

- [ ] Boat ride picture
- [ ] Team page update
- [ ] Hotel discount code/process
- [ ] Welcome Wagon page update

### CODE REVIEW

**Comment by thatdocslady** [comment]
I seem to be having trouble with the conf days in the schedule, they were hard-coded in the schedule.rst page and I tried converting them to variables but I can't find the place where we define those variables... @mxsasha @plaindocs halp? <3

**Comment by mxsasha** [comment]
> I seem to be having trouble with the conf days in the schedule, they were hard-coded in the schedule.rst page and I tried converting them to variables but I can't find the place where we define those variables... @mxsasha @plaindocs halp? <3

They are currently not defined at all, so we need to think about a nice way to add them. The "days" in the current config are not actually real days, a single day there may actually be two calendar days (I'm aware how odd this sounds 😅).

**Comment by plaindocs** [comment]
Yeah, what @mxsasha said. Which is why they are hardcoded. :-D

**Comment by plaindocs** [comment]
Should we just refactor *everything* so that the days are always only one day? :boom:

**Comment by thatdocslady** [comment]
> Should we just refactor _everything_ so that the days are always only one day? 💥

Grumble mumble hard-coding days, if it's not too much trouble I think it's high time we did this properly ;) <3

**Comment by plaindocs** [comment]
Right, But proper isn't obvious. 

image

**Comment by thatdocslady** [comment]
> Right, But proper isn't obvious.
> 
> image

Is this from the Prague website? How do you know without the Netlify preview? I'm lost :(

Should we try and standardize the variables *now* or should I just hard-code and defer it for "later"? @ericholscher @mxsasha votes please :)

**Comment by mxsasha** [comment]
I will look into this tomorrow.

**Comment by ericholscher** [comment]
I think something like this:

```
days:
    - date: Saturday, May 2
      event: Hike
    - date: Sunday, May 3
      event: Writing Day

{% for day in days %}
{{ day.event }}
{% endfor %}
```

But yea, it's annoying that the conf is on 2 days, but we can likely special case that in the one place it's displayed.

**Comment by thatdocslady** [comment]
I'm not emotionally attached to the implementation, just trying to avoid situations where I have to review each topic and modify dates and times because we hard-coded things that could be variables...

File: docs/conf/prague/2020/welcome-wagon.rst [hunk 1]
```diff
@@ -1,15 +1,256 @@
...
+FAQ
+---
+
+.. contents::
+   :local:
+
+Where is everything?
+~~~~~~~~~~~~~~~~~~~~
+
+The conference main stage is in the Auto Klub on the second floor. The unconference
```
Comment (mxsasha) [inline docs/conf/prague/2020/welcome-wagon.rst]: No it's not ;)
Quite a few other Auto Klub references in the rest of this document too.

File: docs/include/conf/prague/visiting.rst [hunk 2]
```diff
@@ -88,11 +88,11 @@ If you need a tax invoice or prefer to pay in cash, there are several reliable l
 Where to stay
 -------------
 
-Our venue Auto Klub is located across the road from the main train station, at the edge of the old town district (Prague 1).
-We recommend booking accommodation in Prague 1 or Prague 2 area, on the west side of the railway station.
+Hotel Olšanka is located in Prague 3, Žižkov district. This neighborhood is close to the city center, and there are plenty of accommodation options in the area for any budget. If you want to stay at the hotel, we have secured a discount code for our attendees. 
 
-We are not working with any of the hotels for attendee accommodations, but there are countless hotels and Airbnb apartments available within walking distance of the venue.
-Just head over to your favorite booking site or use one of these sites to research accommodation:
+Please note that the hotel does not offer accessible rooms, and Write the Docs cannot be held liable for mobility issues outside the conference areas, which are fully accessible.
```
Comment (mxsasha) [inline docs/include/conf/prague/visiting.rst]:
```suggestion
Please note that the hotel does not offer accessible rooms, and Write the Docs is not responsible for mobility issues outside the conference areas, which are fully accessible.
```
Slightly less legalese.

**Comment by mxsasha** [comment]
The day thing is fixed now. Don't know if it's the perfect fix, but it'll do and mean less hardcoding of dates.

File: docs/conf/prague/2020/schedule.rst [hunk 3]
```diff
@@ -81,7 +81,7 @@ We'll also help groups organize dinner plans, so you can continue your conversat
 * **Where**: {{about.venue}}, {{about.mainroom}}
 * **When**: **17:00-20:00**
 
-Monday, May 4
+{{date.day_three.dotw}}, {{date.day_three.date}}
 -------------
```
Comment (ericholscher) [inline docs/conf/prague/2020/schedule.rst]: These likely need to be adopted to be dynamic, as an example:

```suggestion
--{{ "-" * date.day_three.dotw|length }}{{ "-" * date.day_three.date|length }}
```

Comment (thatdocslady) [reply docs/conf/prague/2020/schedule.rst]: @ericholscher should I apply this suggestion? you wrote that it's an example, so I'm not sure? what about other places? maybe @mxsasha can test?

File: docs/conf/prague/2020/team.rst [hunk 4]
```diff
@@ -33,20 +33,20 @@ Sasha Romijn
...
 
-* **Conference chair** - *TBD*
-* **Conference co-chair** - *TBD*
+* **Conference chair** - Mikey Ariel
+* **Conference co-chair** - Samuel Wright
 * **Code of conduct response** - See `code of conduct </code-of-conduct/#reporting-and-contact-information>`_ page
-* **Speaker coordinator** - *TBD*
+* **Speaker coordinator** - Samuel Wright
 * **Unconference coordinator** - *TBD*
 * **Lightning Talk coordinator** - *TBD*
```
Comment (thatdocslady) [inline docs/conf/prague/2020/team.rst]:
```suggestion
* **Lightning Talk coordinator** - Paul Adams
```

File: docs/conf/prague/2020/team.rst [hunk 5]
```diff
@@ -33,20 +33,20 @@ Sasha Romijn
...
-* **Infrastructure coordinator** - *TBD*
-* **Swag coordinator** - *TBD*
-* **Communication coordinator** - *TBD*
-* **Explore Portland coordinators** - *TBD*
+* **Boat ride coordinator** - Mikey Ariel
+* **Infrastructure coordinator** - Sasha Romijn & Samuel Wright
+* **Swag coordinator** - Mikey Ariel
+* **Communication coordinator** - Mikey Ariel
+* **Explore Prague coordinators** - *TBD*
 * **Job Fair coordinator** - *TBD*
```
Comment (thatdocslady) [inline docs/conf/prague/2020/team.rst]:
```suggestion
* **Job Fair coordinator** - Sasha Romijn
```

File: docs/conf/prague/2020/team.rst [hunk 6]
```diff
@@ -33,20 +33,20 @@ Sasha Romijn
...
 * **Welcome Wagon coordinators** - *TBD*
-* **Sponsorship coordinator** - *TBD*
+* **Sponsorship coordinator** - Sasha Romijn
 * **Writing Day coordinator** - *TBD*
```
Comment (thatdocslady) [inline docs/conf/prague/2020/team.rst]:
```suggestion
* **Writing Day coordinator** - Chris Ward
```

File: docs/conf/prague/2020/welcome-wagon.rst [hunk 7]
```diff
@@ -1,15 +1,256 @@
...
+Hi! If you're new to Write the Docs, you're in the right place. The Welcome Wagon is here to help out new attendees, 
+with both the information on this page, and events we run during the conference.
 
-Schedule
+On this page you'll find important stuff that will help you navigate the conference like a pro,
+make you feel more at home, and help you to manage the constant flow of information.
+
+.. image:: /_static/conf/images/pics/welcome-wagon.jpg
+
+Dururing the conference, we'll connect you with people - both veterans and other first-timers.
```
Comment (thatdocslady) [inline docs/conf/prague/2020/welcome-wagon.rst]:
```suggestion
During the conference, we'll connect you with people - both veterans and other first-timers.
```

File: docs/conf/prague/2020/welcome-wagon.rst [hunk 8]
```diff
@@ -1,15 +1,256 @@
...
+Hi - we're Beth and Daniel, your Welcome Wagon coordinators! We're really glad you're coming to the {{year}} Write the Docs conference.
+
+If you need a hand at all, we're here to help.
+
+Welcome Wagon events
+--------------------
+
+**Write the Docs Introduction**
+
+*{{date.day_two.dotw}}, {{date.day_two.date}} at 17:00 in the downstairs foyer (near the cloakroom) at Auto-Klub*
```
Comment (thatdocslady) [inline docs/conf/prague/2020/welcome-wagon.rst]:
```suggestion
*{{date.day_two.dotw}}, {{date.day_two.date}} at 17:00 in the downstairs foyer (near the cloakroom)*
```

File: docs/conf/prague/2020/welcome-wagon.rst [hunk 9]
```diff
@@ -1,15 +1,256 @@
...
+
+Join us for an informal introduction to Write the Docs, to the Welcome Wagon, and to other first-time conference attendees. We'll pass on some information about the conference specifically for first-timers and give everyone a chance to meet someone new before we join the opening reception.
+
+**Venue Tours**
+
+* *{{date.day_two.dotw}}, {{date.day_two.date}} at 17:30 starting near the registration desk at Auto-Klub*
```
Comment (thatdocslady) [inline docs/conf/prague/2020/welcome-wagon.rst]:
```suggestion
* *{{date.day_two.dotw}}, {{date.day_two.date}} at 17:30 starting near the registration desk*
```

File: docs/conf/prague/2020/welcome-wagon.rst [hunk 10]
```diff
@@ -1,15 +1,256 @@
...
+* *{{date.day_three.dotw}}, {{date.day_three.date}} at 9:15 starting near the registration desk at Auto-Klub*
```
Comment (thatdocslady) [inline docs/conf/prague/2020/welcome-wagon.rst]:
```suggestion
* *{{date.day_three.dotw}}, {{date.day_three.date}} at 9:15 starting near the registration desk*
```

File: docs/conf/prague/2020/welcome-wagon.rst (hunk 1 above)
Comment (thatdocslady) [inline docs/conf/prague/2020/welcome-wagon.rst]:
```suggestion
The conference main stage is in the Congress Hall on the second floor. The unconference
```

File: docs/conf/prague/2020/welcome-wagon.rst [hunk 11]
```diff
@@ -1,15 +1,256 @@
...
+---
+
+.. contents::
+   :local:
+
+Where is everything?
+~~~~~~~~~~~~~~~~~~~~
+
+The conference main stage is in the Auto Klub on the second floor. The unconference
+takes place directly across the foyer in the adjoining room.
```
Comment (thatdocslady) [inline docs/conf/prague/2020/welcome-wagon.rst]:
```suggestion
takes place in the Bistro and adjoining rooms, on the left-hand side of the foyer if you stand in front of the Congress Hall.
```

File: docs/conf/prague/2020/welcome-wagon.rst [hunk 12]
```diff
@@ -1,15 +1,256 @@
...
+
+If you are joining in the boat tour on Saturday, you'll meet the other folks
+at the Prague Boats pier no. 5.
+
+How should I get around?
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  Prague is a very walk-able town. Most of the events are
+   within walking distance of the conference venue.
```
Comment (thatdocslady) [inline docs/conf/prague/2020/welcome-wagon.rst]:
```suggestion
   within walking distance or a short public transportation ride away from the conference venue.
```

File: docs/conf/prague/2020/welcome-wagon.rst [hunk 13]
```diff
@@ -1,15 +1,256 @@
...
+   were here we had a heat wave, and last year we got rained on, so you
+   never know! Layering is usually the way to go.
+
+What will I eat?
+~~~~~~~~~~~~~~~~
+
+-  Drinks and food are provided for you during conference days, so you can
+   focus on the talks and meeting people and don't have to worry about
+   where to get your morning coffee.
+-  Coffee, tea, and water are always available in the Auto Klub.
```
Comment (thatdocslady) [inline docs/conf/prague/2020/welcome-wagon.rst]:
```suggestion
-  Coffee, tea, and water are always available in the venue.
```

File: docs/conf/prague/2020/welcome-wagon.rst [hunk 14]
```diff
@@ -1,15 +1,256 @@
...
+   Bring a water bottle to make it easier for you to stay hydrated.
+-  Food is provided on Sunday, Monday, and Tuesday in the Auto Klub. There
```
Comment (thatdocslady) [inline docs/conf/prague/2020/welcome-wagon.rst]:
```suggestion
-  Food is provided on Sunday, Monday, and Tuesday in the conference venue. There
```

File: docs/conf/prague/2020/welcome-wagon.rst [hunk 15]
```diff
@@ -1,15 +1,256 @@
...
+   is a light breakfast, a solid lunch, and snacks on each conference
+   day.
+-  On Saturday and in the evenings on Monday and Tuesday, explore the food
+   options in Prague. Invite someone you just met to join you! If you are
+   invited to dinner, say yes! Making connections over dinner is a great way
+   to get to know more people.
+-  If you need grocery items, there is a BILA supermarket at the Prague train station
+   across the park from Auto Klub.
```
Comment (thatdocslady) [inline docs/conf/prague/2020/welcome-wagon.rst]:
```suggestion
-  If you need grocery items, there is an Albert supermarket on Ondříčkova 39, 150m west of the venue.
```

File: docs/conf/prague/2020/welcome-wagon.rst [hunk 16]
```diff
@@ -1,15 +1,256 @@
...
+
+Where should I sit?
+~~~~~~~~~~~~~~~~~~~
+
+-  The Auto Klub will have round tables next to the main stage
```
Comment (thatdocslady) [inline docs/conf/prague/2020/welcome-wagon.rst]:
```suggestion
-  The Congress Hall has round tables next to the main stage
```

File: docs/conf/prague/2020/welcome-wagon.rst [hunk 17]
```diff
@@ -1,15 +1,256 @@
...
+   and rows of seats behind them. There will also be seats in the mezzanine level.
+-  There are no reserved seats; feel free to sit anywhere.
+-  If you can, show up early to the conference each morning to grab a
+   seat at one of the round tables. Introducing yourself to your
+   neighbors is one of the easiest way to meet people.
+
+What should I do during the talks?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+`Conference talks at Auto Klub </conf/{{shortcode}}/{{year}}/speakers/>`__:
```
Comment (thatdocslady) [inline docs/conf/prague/2020/welcome-wagon.rst]:
```suggestion
`Conference talks </conf/{{shortcode}}/{{year}}/speakers/>`__:
```

File: docs/conf/prague/2020/welcome-wagon.rst [hunk 18]
```diff
@@ -1,15 +1,256 @@
...
+
+-  The time between talks is for meeting your colleagues or taking a
+   break. During the talks, listen and take in as much as you can.
+-  There is a lot of great information at this conference, but don't
+   worry if you miss something! All talks are videotaped, so you can
```
Comment (thatdocslady) [inline docs/conf/prague/2020/welcome-wagon.rst]:
```suggestion
   worry if you miss something! All talks are recorded, so you can
```

File: docs/conf/prague/2020/schedule.rst (hunk 3 above)
Comment (mxsasha) [reply docs/conf/prague/2020/schedule.rst]: I am handling this.

**Comment by plaindocs** [comment]
This one is replaced by #1251

**Pull request closed without merging**